css/modals/invite/_invite.scss

```diff
@@ -63,6 +63,14 @@
             flex: 1;
             margin-right: 10px;
         }
+
+        /**
+         * Overwrite all AtlasKit styles in order to align the button text
+         * to the left.
+         */
+        .efkQxJ, .liMDSM, .sFXRd, .jKlmrw, .jSKXCE, .ctDuVH {
+            text-align: left;
+        }
```

Comment:
The button trigger will always align the text in the center and all those different styles are used on hover, click, focus, etc. It wasn't possible to just add :hover and :focus. I just hope that this is going away soon, because we're moving this dropdown to another dialog and won't have the same constraints there.

Reply:
Are these all for buttons? Some components take in a className prop that might be useable for overriding styles.

Reply:
Thanks! Did work with className indeed!